jina/serve/runtimes/gateway/grpc/gateway.py

```diff
@@ -36,15 +36,17 @@ def __init__(
         self.grpc_server_options = grpc_server_options
         self.ssl_keyfile = ssl_keyfile
         self.ssl_certfile = ssl_certfile
-        self.server = grpc.aio.server(
-            options=_get_grpc_server_options(self.grpc_server_options)
-        )
         self.health_servicer = health.HealthServicer(experimental_non_blocking=True)
 
     async def setup_server(self):
         """
         setup GRPC server
         """
+        self.server = grpc.aio.server(
```

Comment:
Can't create the server in `init` because the opentelmetry dependencies are not available at the `init` level. These are set by the `inject_dependencies` method.